File: Leetcode/Daily/07-2025/Java/01.java
// Problem: 3330. Find the Original Typed String I (Easy)
// Link: https://leetcode.com/problems/find-the-original-typed-string-i

class Solution {
    public int possibleStringCount(String word) {
        int ans = 1;
        int n = word.length();

        for (int i = 1; i < n; i++) {
            if (word.charAt(i) == word.charAt(i-1)) ans++;
        }

        return ans;
    }
}

File: Leetcode/Daily/07-2025/TypeScript/01.ts
// Problem: 3330. Find the Original Typed String I (Easy)
// Link: https://leetcode.com/problems/find-the-original-typed-string-i

function possibleStringCount(word: string): number {
    let ans = 1;
    const n = word.length;

    for (let i = 1; i < n; i++) {
        if (word[i] === word[i-1]) ans++;
    }

    return ans;
};
